### DISCUSSION TITLE
Possible to include the Unifi Protect Doorbell messages?

### DISCUSSION THREAD

**Discussion by danielmtnz** [koush/scrypted#360] [Ideas]
Is it possible to have the Unifi Protect Doorbell messages appear as buttons? I see there is a button to perform actions when the doorbell button is pressed, would like to use a pre-configured message to display when it is pressed. 

Thank you.

**Comment by koush** [comment]
You can use the dummy switch plugin, and then have the dummy switch trigger an automation in scrypted that sends a message.
